google/cloud/logging_v2/entries.py

```diff
@@ -165,8 +167,13 @@ def from_api_repr(cls, resource, client, *, loggers=None):
         logger_fullname = resource["logName"]
         logger = loggers.get(logger_fullname)
         if logger is None:
-            logger_name = logger_name_from_path(logger_fullname)
-            logger = loggers[logger_fullname] = client.logger(logger_name)
+            # attempt to create a logger if possible
+            try:
+                logger_name = logger_name_from_path(logger_fullname, client.project)
+                logger = loggers[logger_fullname] = client.logger(logger_name)
+            except ValueError:
+                # log name is not scoped to a project. Leave logger as None
```

Comment:
Is this comment true? Seems that logger is initialized with value returned from `loggers.get(logger_fullname)` call...

Reply:
`loggers.get(logger_fullname)` will return `None` if the loggers dict doesn't contain `logger_fullname`

If it's `None`. Lines 171 to 173 will try to contruct a new logger associated with `logger_name`, which will work if `logger_name` is a project id. If it's an organization or a folder, it will throw an exception. In that case, we can just leave the logger as empty and move on